src/lib/structures/profiling.ml

```diff
@@ -536,7 +533,8 @@ let print_instances_generation forced _steps fmt _timers =
         ) insts
     in
     List.iter
-      (fun (name, i, card, r) ->
+      (let fprintf = Format.fprintf in
```

Comment:
Why not `let open Format in` ?

Reply:
it's a matter of taste. If you need only one function in a module, why import it completely? It's pity that Ocaml cannot open partially a module...